pkgs/tools/admin/hedgedoc-cli/default.nix

```diff
@@ -14,16 +14,16 @@ stdenv.mkDerivation {
     sha256 = "uz+lkRRUTRr8WR295esNEbgjlZ/Em7mBk6Nx0BWLfg4=";
   };
 
-  buildInputs = [
-    wget
-    jq
-    curl
+  nativeBuildInputs = [
+    makeWrapper
   ];
 
   installPhase = ''
     runHook preInstall
-    install -Dm0755 -t $out/bin $src/bin/codimd
-    ln -s $out/bin/codimd $out/bin/hedgedoc-cli
+    makeWrapper $src/bin/codimd $out/bin/codimd \
+      --prefix PATH : ${lib.makeBinPath [ jq wget curl ]}
```

Comment:
```suggestion
    wrapProgram $src/bin/codimd \
      --prefix PATH : ${lib.makeBinPath [ jq wget curl ]}
```

Reply:
But then...

```
~/C/N/nixpkgs   fix/hedgedoc-cli-fix-shell-script  nix shell .#hedgedoc-cli                                                                                                                                                                                                                                                                                                                                                          
error: builder for '/nix/store/hi3w5xdppax8ql2h8gcrrmgrnn2m6x6h-hedgedoc-cli-1.0.drv' failed with exit code 1;
       last 10 log lines:
       > installing
       >
       > Builder called die: Cannot wrap '/nix/store/6ryw05pxfh69in3dp8vapw1vlg358ygf-hedgedoc-cli-1.0/bin/codimd' because it is not an executable file
       > Backtrace:
       > 7 assertExecutable /nix/store/jk1bi0mby6ninq6c966k676kznsc4fb2-hook/nix-support/setup-hook
       > 213 wrapProgramShell /nix/store/jk1bi0mby6ninq6c966k676kznsc4fb2-hook/nix-support/setup-hook
       > 208 wrapProgram /nix/store/jk1bi0mby6ninq6c966k676kznsc4fb2-hook/nix-support/setup-hook
       > 1389 genericBuild /nix/store/43215b010yhnvpmr169abg324zq1l0sr-stdenv-linux/setup
       > 2 main /nix/store/9krlzvny65gdc8s7kpb6lkx8cd02c25b-default-builder.sh
       >
       For full logs, run 'nix log /nix/store/hi3w5xdppax8ql2h8gcrrmgrnn2m6x6h-hedgedoc-cli-1.0.drv'.
 ```

Reply:
Either that file does not exist and it still need to be copied, then that should be done. If the file ist just missing the x bit add ``chmod +x $out/bin/codimd``